docs/source/usingmame/defaultkeys.rst

```diff
@@ -1,106 +1,476 @@
++------------------------------------+-----------------------------------------------------------------------------------------------------------+
+| **Insert**                         | | Fast forward. While held, runs game with throttling disabled and with the maximum frameskip.            |
+|                                    | | (*Windows non-SDL MAME only*)                                                                           |
++------------------------------------+-----------------------------------------------------------------------------------------------------------+
+| **Page DN**                        | | Fast forward. While held, runs game with throttling disabled and with the maximum frameskip.            |
+|                                    | | (*SDL MAME only*)                                                                                       |
++------------------------------------+-----------------------------------------------------------------------------------------------------------+
+| **Alt+ENTER**                      | Toggles between full-screen and windowed mode.                                                            |
++------------------------------------+-----------------------------------------------------------------------------------------------------------+
+| **Scroll Lock**                    | Default mapping for the **uimodekey**.                                                                    |
+|                                    |                                                                                                           |
+|                                    | | This key allows users to disable and enable the emulated keyboard in machines that require it.          |
+|                                    | | All emulations which require emulated keyboards will start in that mode and you can only access         |
+|                                    | | the internal UI (hitting TAB) by first hitting this key. You can change the initial status of the       |
+|                                    | | emulated keyboard as presented upon start by using **-ui_active**.                                      |
+|                                    |                                                                                                           |
+|                                    | FIXME: Add link back to -ui_active                                                                        |
++------------------------------------+-----------------------------------------------------------------------------------------------------------+
+| **Escape**                         | Exits emulator.                                                                                           |
++------------------------------------+-----------------------------------------------------------------------------------------------------------+
```

Comment:
Are you going to describe the other UI controls?  For example the navigation directions (used on most menus), focus next/previous (used on the system and software selection menus), and the other controls used on the system/software selection menus?

Reply:
Covered now for the most part. There are a few elements that aren't obvious yet though. We'll talk about those when I submit the next push.